test/org/opensolaris/opengrok/condition/ConditionalRunRule.java

```diff
@@ -52,14 +51,14 @@
     public Statement apply(Statement aStatement, Description aDescription) {
         if (hasConditionalIgnoreAnnotationOnMethod(aDescription)) {
             RunCondition condition = getIgnoreConditionOnMethod(aDescription);
-            if (!condition.isSatisfied()) {
+            if (!condition.isForcedOrSatisfied()) {
```

Comment:
why this needs to be changed ?

Reply:
`isSatisfied()` is not generic because it does not take into account `-Djunit-force-all` property. Therefore, I created separate method `isForcedOrSatisfied` which checks if force property is set and if not then calls `isSatisified()`.